camel/models/base_model.py

```diff
@@ -138,3 +142,83 @@ def stream(self) -> bool:
             bool: Whether the model is in stream mode.
         """
         return False
+
+
+class ModelManager:
+    def __init__(
```

Comment:
I think we can make this class under a new python file called `model_manager.py`

Reply:
Moved to a separate module.